libs/ngx-form-factory/src/lib/fields/field.ts

```diff
@@ -57,6 +63,8 @@ export interface IField<T> extends IBaseField<T>, MatFormFieldDefaultOptions {
   hint?: string;
   placeholder?: string;
   class?: string | string[];
+  suffix?: ISuffixPrefixOptions;
```

Comment:
suffix and prefix should be strings, don't need an interface and only be added to ITextOptions

Reply:
Done